main.py

```diff
@@ -203,24 +215,23 @@ def collision():
 	# Event loop
 	for event in pygame.event.get():
 		
-		if event.type == pygame.QUIT or event.type == pygame.KEYDOWN:
+		if event.type == pygame.QUIT or event.type == pygame.KEYDOWN:		# bug to be fixed

```

Comment:
If you know the existence of a bug, then either fix it or keep this outside 'main' branch. For eg, keep it on your branch and merge it only after fixing this!
I will approve this just for one time!

Reply:
Ok, senior. It is fixed now.